### ISSUE TITLE
There is no VRR switch in the settings

### ISSUE THREAD

**Issue by fisuri** [KyleGospo/gnome-vrr#9]
Installed:

GPU - rx5700xt

- mutter | 44.3.vrr.16-1
- xorg-x11-server-Xwayland | 23.1.2-1
- gnome-control-center | 44.3.vrr.4-1

**Comment by KyleGospo** [comment]
For the toggle, make sure you have this in `/etc/environment`:
`MUTTER_DEBUG_FORCE_KMS_MODE=simple`

And also run the following in case the defaults didn't take:
`gsettings set org.gnome.mutter experimental-features "['variable-refresh-rate']"`

**Comment by peregr1nefalco** [comment]
Sorry for necroing the thread, I've installed the packages via Copr:


`[root@shaheen ~]# dnf list installed | grep kylegospo`
`gnome-control-center.x86_64  44.3.vrr.4-1.fc38  @copr:copr.fedorainfracloud.org:kylegospo:gnome-vrr`
`gnome-control-center-filesystem.noarch  44.3.vrr.4-1.fc38  @copr:copr.fedorainfracloud.org:kylegospo:gnome-vrr`   
`mutter.x86_64  44.3.vrr.17-1.fc38  @copr:copr.fedorainfracloud.org:kylegospo:gnome-vrr`     
`mutter-common.noarch  44.3.vrr.17-1.fc38  @copr:copr.fedorainfracloud.org:kylegospo:gnome-vrr` 
`xorg-x11-server-Xwayland.x86_64  23.1.2-1.fc38  @copr:copr.fedorainfracloud.org:kylegospo:gnome-vrr`

I've also set the required env settings in `/etc/environment` and `gsettings`:

`[root@shaheen ~]# cat /etc/environment`
`MUTTER_DEBUG_FORCE_KMS_MODE=simple`

`[root@shaheen ~]# gsettings get org.gnome.mutter experimental-features`
`['variable-refresh-rate']`

But in my case the switch still does not show up.

Any pointers on what to check? Thanks!
